docs/index.html

```diff
@@ -0,0 +1,89 @@
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" href='./styles.css' />
+        <link rel="stylesheet" href='https://cdn01.boxcdn.net/platform/preview/1.58.2/en-US/preview.css' />
+        <link rel="stylesheet" href='https://unpkg.com/box-annotations@3.3.0/lib/annotations.css' />
+        <script src='https://cdn01.boxcdn.net/platform/preview/1.58.2/en-US/preview.js'></script>
+        <script src='https://unpkg.com/box-annotations@3.3.0/lib/annotations.js'></script>
```

Comment:
Eventually we can pass in a version, but for now we can just make a commit here when doing a release